### ISSUE TITLE
Mode Endpoint has Changed

### ISSUE THREAD

**Issue by jwillaz** [tchellomello/python-arlo#92]
When changing modes (Armed, Disarmed, etc), a POST is now made to a different endpoint:  

**https://arlo.netgear.com/hmsweb/users/devices/automation/active**

with a payload similar to the following:

{
   "activeAutomations":[
      {
         "deviceId":"xxxxxxxxxx",
         "timestamp":1531211943092,
         "activeModes":[
            "mode1"
         ],
         "activeSchedules":[
         ]
      }
   ]
}

**Comment by tchellomello** [comment]
@Nicxe sorry I've been a little busy lately but hopefully, someone will take this issue.

**Comment by collse** [comment]
Has there been any movement on this - seems like its now affecting the front/back end performance as well

**Comment by nzben** [comment]
I'm picking this up. I'd be keen to understand where you see the data posed @jwillaz. When I post that payload (with my deviceId and current timestamp) I get 

`{u'data': {u'message': u'Failed to update Automation Definitions', u'reason': u'Unable to upload active automation. Please try again', u'error': u'AUTO-5050'}, u'success': False}`

**Comment by teqnet** [comment]
@nzben
Any progress here. My arlo is unknown 50% of the time (every seccond sample)

**Comment by teqnet** [comment]
Update of alarm_control_panel.arlo_t is taking over 10 seconds

**Comment by JesperSkagerberg** [comment]
I am also very interested in the progress. Thank you in advance.

**Comment by zemerick1** [comment]
Would love this to work so I could utilize some automation.
I've ran several basic troubleshooting tasks using pyarlo directly. 

View mode
View available modes
Set mode
View new mode

It is really hit or miss. Sometimes it will return 'none' as the new set mode. sometimes it will return the same mode as the current set mode, indicating it never got changed. Not sure it is the code as much as it is Arlo.

**Comment by nzben** [comment]
I'm not sure if @jwillaz initial issue is correct, because as I posted back in November, I can't get any valid response, nor any valid mode change.

**Comment by zemerick1** [comment]
> I'm not sure if @jwillaz initial issue is correct, because as I posted back in November, I can't get any valid response, nor any valid mode change.

Yeah, I may be in the wrong thread. However, there is an issue where the mode won't change.

**Comment by collse** [comment]
its not only the mode change that is broken, I added another camera yesterday and it doesnt show up either - I think the entire component needs reviewing - unfortunately it seems no one has the time or skill at the moment to address the issues ...
